Presentation.Web/app/components/org/structure/org-unit-migration-table.component.ts

```diff
@@ -0,0 +1,159 @@
+        organizationUuid: string | null = null;
+        root: IOrganizationUnitMigrationRoot;
+        columnTypes = MigrationTableColumnType;
+
+        static $inject: string[] = ["organizationRegistrationsService", "notify", "$state", "$window"];
+        constructor(private readonly organizationRegistrationsService: Services.Organization.IOrganizationRegistrationsService,
+            private readonly notify,
+            private readonly $state,
+            private readonly $window) {
+        }
+
+        $onInit() {
+            if (this.title === null) {
+                console.error("missing migration table attribute: 'title'");
+            }
+            if (this.options === null) {
+                console.error("missing migration table attribute: 'options'");
+            }
+            if (this.configuration === null) {
+                console.error("missing migration table attribute: 'configuration'");
+            }
+            if (this.unitUuid === null) {
+                console.error("missing migration table attribute: 'unitUuid'");
+            }
+            if (this.organizationUuid === null) {
+                console.error("missing migration table attribute: 'organizationId'");
+            }
+
+            this.root = this.options.root;
+        }
+
+        registrationSelected() {
+            const callback = this.options.selectedRegistrationChanged;
+            if (callback !== undefined) {
+                callback();
+            }
+        }
+
+        registrationGroupSelected(root) {
+            const callback = this.options.selectedRegistrationGroupChanged;
+            if (callback !== undefined) {
+                callback(root);
+            }
+        }
+
+        delete(registration: Models.ViewModel.Organization.IOrganizationUnitRegistration) {
+            if (this.options.checkIsBusy()) {
+                    return;
+            }
+            if (!confirm('Er du sikker på, at du vil slette registreringen?')) {
+                return;
+            }
+            this.options.setIsBusy(true);
+
+            registration.selected = true;
+            const request = this.createChangeRequest(registration);
+            registration.selected = false;
+
+            this.organizationRegistrationsService.deleteSelectedRegistrations(this.organizationUuid, this.unitUuid, request)
+                .then(() => {
+                    this.options.refreshData();
+                    this.options.setIsBusy(false);
+                },
+                error => {
+                    console.log(error);
+                    this.notify.addErrorMessage("Failed to deleted the selected unit");
+                    this.options.setIsBusy(false);
+                });
+
+            this.options.setIsBusy(false);
+        }
+
+        navigateTo(id: number) {
+            const url = this.$state.href(this.options.dataRelatedPage, { id: id });
+            this.$window.open(url, "_blank");
```

Comment:
why?.... why not just render a link (sref) like we do in overviews?
This introduces the user of $window which is an additional dependency... no need for this